### ISSUE TITLE
review email templates

### ISSUE THREAD

**Issue by xmonader** [codescalers/cloud4students#600]
![image](https://github.com/user-attachments/assets/f8ae2c49-89a7-4c7b-96f0-974e2f164341)

the email template/rendering needs some love

**Comment by rowangamal** [comment]
I agree, they've been sending me emails calling me poor, bear in mind I have about 2000 TFTs

**Comment by rawdaGastan** [comment]
email body needs to have new lines
